backend/app/infrastructure/routes/user.py

```diff
@@ -316,3 +331,26 @@ def get(self, epoch: int):
                 for user_address, uq_score in uq_scores
             ]
         }
+
+
+@ns.route("/<string:user_address>/winnings/<int:epoch>")
```

Comment:
I suggest adding "raffle" to the slug, e.g. `raffle/winnings`. "Winnings" are very broad and if we will introduce any other type of "winning" it'll get really confusing fast.

Reply:
Changed.